Tensile/KernelWriterAssembly.py

```diff
@@ -756,10 +756,7 @@ def initKernel(self, kernel, tPA, tPB ):
       kernel["LocalWriteUseSgprB"] = False # Requires DirectToLdsB
 
     # set up useAtomicAdd
-    # limit to f32 + BufferStore + VAW=1 only
-    # applicable as long as ComputeDataType is single, but limited to _GlobalAccumulation=SingleBuffer so far
-    self.useAtomicAdd = self.asmCaps["HasAtomicAdd"] and (kernel["_GlobalAccumulation"] == 'SingleBuffer') and \
-                        kernel["BufferStore"] and kernel["VectorAtomicWidth"] == 1
+    self.useAtomicAdd = kernel["GlobalSplitUAtomicAdd"]
```

Comment:
I believe we still need to check asm cap.
AtomicAdd is added from gfx908.
Without checking asm cap, it will cause an issue with older arch.

This part is for GSU only.
As long as I know, atomic add is used for StoreCInUnroll.
Strictly speaking, we need asm cap check for StoreCInUnroll as well.
However, it does not make difference because StoreCInUnroll requires MFMA and MFMA is added from gfx908.
So, I do not think we actually need asm cap check for StoreCInUnroll.

Reply:
It looks like the option "AtomicAddC" is not specific to StoreCInUnroll.
It might be possible to enable this option on 906 and causes an error....
I do not have a plan to update AtomicAddC very soon, but I can add asm cap check when I have a chance.

Reply:
If you want, I can quickly add asm cap check for AtomicAddC in this PR (or a new PR).
The change is very easy.

Reply:
Oh ok, if StoreCInUnroll is behind a asm cap check then that should be fine. Thanks!